gammapy/datasets/make.py

```diff
@@ -209,3 +211,128 @@ def make_test_observation_table(observatory_name, n_obs, debug=False):
     obs_table['MUON_EFFICIENCY'] = muon_efficiency
 
     return obs_table
+
+
+def make_test_bg_cube_model(detx_min=Angle(-10., 'degree'),
```

Comment:
Maybe use `detx_range`, `dety_range` and `energy_band` as parameters to have fewer parameters (but same functionality)?

`index` -> `spectral_index`